secp256k1-sys/README.md

```diff
@@ -1,21 +1,25 @@
-secp256k1-sys
-=============
+# Secp256k1-sys (FFI for libsecp256k1)
 
+[![Build Status](https://travis-ci.org/rust-bitcoin/rust-secp256k1.png?branch=master)](https://travis-ci.org/rust-bitcoin/rust-secp256k1)
 
-This crate provides Rust definitions for the FFI structures and methods.
+[Full documentation](https://docs.rs/secp256k1_sys/)
+
+This is the Foreign Function Interface (FFI) to the [libsecp256k1](https://github.com/bitcoin-core/secp256k1) library.
```

Comment:
This sounds a bit strange the library is not FFI. The library just provides low-level bindings to the C FFI exposed by `libsecp256k1`